src/resultsdisassemblypage.cpp

```diff
@@ -112,19 +137,57 @@ void ResultsDisassemblyPage::showDisassembly()
     // Show empty tab when selected symbol is not valid
     if (m_curSymbol.symbol.isEmpty()) {
         clear();
-        return;
     }
 
     // Call objdump with arguments: addresses range and binary file
     QString processName = m_objdump;
     QStringList arguments;
     arguments << QLatin1String("-d") << QLatin1String("--start-address") << QLatin1String("0x") + QString::number(m_curSymbol.relAddr, 16) <<
             QLatin1String("--stop-address") << QLatin1String("0x") + QString::number(m_curSymbol.relAddr + m_curSymbol.size, 16) <<
-            m_curSymbol.path;
+            m_curAppPath;
 
     showDisassembly(processName, arguments);
 }
 
+QByteArray ResultsDisassemblyPage::processDisassemblyGenRun(QString processName, QStringList arguments)
+{
+    QByteArray processOutput = QByteArray();
+    if (m_curSymbol.symbol.isEmpty()) {
+        processOutput = "Empty symbol ?? is selected";
```

Comment:
needs to be translated, repeats below - make the error a QString instead of a QBA and split it out from the QBA of the objdump run

```
struct DisassemblyOutput
{
    QByteArray output;
    QString errorMessage;
    explicit operator bool() const { return errorMessage.isEmpty(); }
    static DisassemblyOutput fromProcess(const QString &processName, const QStringList &arguments);
};
```

Reply:
use a KMessageBox to display the error message then